RocketChat/Rocket.Chat.ReactNative
# fix: starting a new thread from an attachment

## Proposed changes

When starting a new thread from an attachment and it is expected appear the attachment name as the header of the reply. But this wasn't happening as you can see on the screenshots

## Issue(s)	


## How to test or reproduce

- Open a channel
- Send any type of attachment without msg
- Send a new message
- Reply in thread (Send to channel enabled) to that Attachment that isn’t a thread yet.
- After you sent the message, should appear the header (attachment name) with the message that you sent

## Screenshots

#### Before

https://github.com/RocketChat/Rocket.Chat.ReactNative/assets/47038980/ae9132b3-d996-4a15-b54d-05677541df0c


#### After

https://github.com/RocketChat/Rocket.Chat.ReactNative/assets/47038980/003f2eec-887f-4baa-899f-2e4f4f5fbbda




## Types of changes




## Checklist



## Further comments

[TC-1021]

[TC-1021]: https://rocketchat.atlassian.net/browse/TC-1021?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
